### ISSUE TITLE
Running pretrained model and Generate.ipynb giving 0 second audio outputs

### ISSUE THREAD

**Issue by nishkakhendry** [facebookresearch/music-translation#20]
I am working on Ubuntu 20.04 with jupyter notebooks on my laptop. As I don't have a graphics card, I changed the decoder definition from NVWavenetGenerator to WavenetGenerator and removed .cuda() from the code to make it runnable locally. The code runs without a problem but the output is a bunch of 0 second audio clips. I have tried debugging by placing disp() in the forward of the encoder and decoder but have not yet fixed the problem. Any help or ideas would be appreciated. Thanks in advance!
![image](https://user-images.githubusercontent.com/54926737/126982534-0201430c-0d87-44b2-81fe-1c5ed89b4985.png)

**Comment by DanileRond** [comment]
I got the same problem. Anyone solved it?

**Comment by xistva02** [comment]
Yes. This happens when you use WavenetGenerator decoder (WavenetGenerator) but keep the parameters of NVWavenetGenerator from the ipython script. Try to modify decoder from "decoder = NVWavenetGenerator(decoder, rate // 20, batch_size, 3)" to "decoder = WavenetGenerator(decoder, batch_size, wav_freq=rate)". The inference is much slower but it seems to work without needing of wavenet compilation via setup.py or make.
